google-cloud-bigtable/src/test/java/com/google/cloud/bigtable/data/v2/stub/metrics/MetricsTracerTest.java

```diff
@@ -474,4 +480,22 @@ public Object answer(InvocationOnMock invocation) {
   private static <T> StreamObserver<T> anyObserver(Class<T> returnType) {
     return (StreamObserver<T>) any(returnType);
   }
+
+  private class ReverseErrorsConverter
+      extends UnaryCallable<BulkMutation, MutateRowsAttemptResult> {
+
+    private final UnaryCallable<BulkMutation, Void> innerCallable;
+
+    ReverseErrorsConverter(UnaryCallable<BulkMutation, Void> callable) {
+      this.innerCallable = callable;
+    }
+
+    @Override
+    public ApiFuture<MutateRowsAttemptResult> futureCall(
+        BulkMutation request, ApiCallContext context) {
+      ApiFuture<Void> future = innerCallable.futureCall(request, context);
+      return ApiFutures.transform(
+          future, result -> new MutateRowsAttemptResult(), MoreExecutors.directExecutor());
+    }
+  }
```

Comment:
I dont understand what this is doing

Reply:
Please don't do that, instead please make the private callable public and use it.

I can think of 2 ways to make it public:

1. mark the method as package private. Then add a helper class in `src/test/java/com/google/cloud/bigtable/data/v2/stub` that has a public method to extract the callable from a stub ie:
```java
public class StubPrivateAccessor {
  public static UnaryableCallable<...> getMutateRowsBaseCallable(EnhancedBigtableStub s) {
     return s.mutateRowsStub()
  }
}
```

2. add a public getter to the EnhancedStub for the callable and mark it as InternalApi